operate/client/src/App/ProcessInstance/BottomPanel/Variables/v2/index.tsx

```diff
@@ -25,20 +23,33 @@ import {Footer} from '../Footer';
 import {Skeleton} from '../Skeleton';
 import {useDisplayStatus} from 'modules/hooks/variables';
 import {useNewScopeIdForFlowNode} from 'modules/hooks/modifications';
+import {usePermissions} from 'modules/queries/permissions/usePermissions';
+import {flowNodeMetaDataStore} from 'modules/stores/flowNodeMetaData';
+import {useIsProcessInstanceRunning} from 'modules/queries/processInstance/useIsProcessInstanceRunning';
 
 type Props = {
   isVariableModificationAllowed?: boolean;
 };
 
+type FooterVariant =
+  | 'initial'
+  | 'disabled'
+  | 'add-variable'
+  | 'pending-variable';
```

Comment:
It's a bit harder to read but then it's easier to get them in sync

```suggestion
type FooterVariant = React.ComponentProps<typeof Footer>['variant'];
```